libs/libtirpc/patches/800-bzero-to-memset.patch

```diff
@@ -0,0 +1,11 @@
+--- a/src/des_impl.c
++++ b/src/des_impl.c
+@@ -588,7 +588,7 @@ _des_crypt (char *buf, unsigned len, str
+     }
+   tin0 = tin1 = tout0 = tout1 = xor0 = xor1 = 0;
+   tbuf[0] = tbuf[1] = 0;
+-  __bzero (schedule, sizeof (schedule));
++  memset (schedule, 0, sizeof (schedule));
```

Comment:
After a little reading I found that you can get caught out with this trick, in that some compilers still optimise _explicit_bzero_ out and then you are back to the same situation that you are suggestion could be a security issue. From reading, it is suggested that testing has to be done to make sure that the compiler has not optimised it out, it doesn't say how to test this. I feel this is currently outside of my knowledge and ability.